apps/assisted_tagging/schema.py

```diff
@@ -104,19 +103,7 @@ def get_draft_entry_qs(info):  # TODO use dataloader
     qs = DraftEntry.objects.filter(project=info.context.active_project).order_by('page', 'text_order')
     if PP.check_permission(info, PP.Permission.VIEW_ENTRY):
         return qs.prefetch_related(
-            Prefetch(
-                'predictions',
-                queryset=AssistedTaggingPrediction.objects.filter(is_selected=True).order_by('id'),
-            ),
-            Prefetch(
-                'related_geoareas',
-                queryset=get_geo_area_queryset_for_project_geo_area_type().order_by('id'),
-            ),
             'predictions__model_version',
```

Comment:
Let's move this to new dataloader logic

Reply:
we have already  added logic  in geo/loaders